### PR TITLE
Added a Dockerfile.demo to load the data. This is not working at the …

### PR DESCRIPTION
**Pull request by mbucknell**
…moment. But that restore command does work when I run it by hand

### CODE REVIEW

**Comment by mbucknell** [comment]
I was not able to get the pg_restore to work on a gzipped archive file (kept complaining about a missing toc). I have not yet but the tar file in artifactory.

File: liquibase/changeLogs/wqp/changeLog.yml [hunk 1]
```diff
@@ -8,25 +8,31 @@ databaseChangeLog:
       relativeToChangelogFile: true
 
   - include:
+      context: dev,qa,prod,ci
```
Comment (dsteinich) [inline liquibase/changeLogs/wqp/changeLog.yml]: This seems a little dirty. Will probably want to handle differently in the future.

File: liquibase/Dockerfile.demo [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM alpine
```
Comment (dsteinich) [inline liquibase/Dockerfile.demo]: I don't like another container just to run pg_restore. I'm imagining a different JIRA to refactor the entire Docker portion to more of a staged build and docker save & load.

File: docker-compose.yml [hunk 3]
```diff
@@ -28,3 +28,43 @@ services:
       - ./liquibase/changeLogs:/opt/liquibase/workspace
       - ./liquibase/logs:/opt/liquibase/logs
       - ./liquibase/dbInit:/docker-entrypoint-initdb.d
+
+  liquibase_demo:
```
Comment (dsteinich) [inline docker-compose.yml]: See comment in Dockerfile.demo.

**Pull request merged**